api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -360,6 +371,10 @@ message Bootstrap {
   // Envoy only supports ListenerManager for this field and Envoy Mobile
   // supports ApiListenerManager.
   core.v3.TypedExtensionConfig listener_manager = 37;
+
+  // Optional field to set the application logs format. If this field is set, it will override
+  // the default log format, or the format that is specified by the CLI option ``--log-foramt``.
```

Comment:
done

Reply:
Yeah I understand your point. I'm not familiar with other CLI options that also have a similar configurable feature, do you?
I think there's a good point for this override, specifically because if you want to support full JSON experience then you would need to use both ``--log-format`` as well as the ``application_log_format`` - I elaborated the full details in the ``application_logs`` docs. Wdyt?